config/auth.php

```
@@ -15,7 +15,7 @@
 	|
 	*/
 
-	'driver' => 'eloquent',
+	'driver' => env('AUTH_DRIVER', 'eloquent'),
```

I fail to see the need to have this setting as ENV variables rather than hardcoded.

It doesn't need to be dynamic based on environment or kept as sensitive information.

I could see DB_DRIVER, FILESYSTEM_DRIVER, and MAIL_DRIVER changing based on environment.